gradio/routes.py

```diff
@@ -4,6 +4,11 @@
 from __future__ import annotations
 
 import asyncio
+import sys
+if sys.version_info >= (3, 9):
+    from importlib.resources import files
+else:
+    from importlib_resources import files
```

Comment:
Hmm we'll need to add `importlib_resources` as a dependency if its not available in python 3.8

Reply:
good catch